build.gradle

```diff
@@ -82,7 +82,7 @@ ext {
 
             retrolambda: [
                     config: '2.5.1',
-                    plugin: '3.5.0'
+                    plugin: '3.7.1'
```

Comment:
Do we still need this? I don't think so... right?

Reply:
removed.